packages/rust-bindings/package.json

```diff
@@ -17,8 +17,6 @@
         "/pkg/**/concordium_rust_bindings*"
     ],
     "scripts": {
-        "lint": "cargo clippy",
-        "lint-fix": "cargo +nightly-2022-06-09-x86_64-unknown-linux-gnu fmt",
```

Comment:
Why remove these? Wouldn't these still be useful to apply the formatting expected by the CI flow?

Also, align the version used for fmt with the new version in the CI (if we still need these scripts).

Reply:
They already have separate jobs in the CI
(The version used in the fmt job should probably be updated though, to nightly-2023-04-01, can you do that here @rasmus-kirk :pleading_face: )

Reply:
Yeah, I'll do that

Reply:
Done, take a look